internal/cli/up.go

```diff
@@ -34,39 +39,43 @@ func (c *upCmd) register() *cobra.Command {
 func (c *upCmd) run(args []string) error {
 	span := opentracing.StartSpan("Up")
 	defer span.Finish()
-	ctx := opentracing.ContextWithSpan(context.Background(), span)
-	proc, err := tiltd_server.RunDaemon(ctx)
+	ctx := logger.WithLogger(
+		opentracing.ContextWithSpan(context.Background(), span),
+		logger.NewLogger(logLevel(), os.Stdout),
+	)
+
+	logOutput("Starting Tilt…")
+
+	tf, err := tiltfile.Load("Tiltfile")
 	if err != nil {
 		return err
 	}
 
-	defer func() {
-		err := proc.Kill()
-		if err != nil {
-			log.Fatalf("failed to shut down daemon: %v", err)
-		}
-	}()
-
-	dCli, err := tiltd_client.NewDaemonClient()
+	serviceName := args[0]
+	protoServices, err := tf.GetServiceConfig(serviceName)
```

Comment:
+1 for changing GetServiceConfig to return domain objects

Reply:
will do in another diff, gonna land this asap so i don't get more weird merge shit